spot-client/src/index.js

```diff
@@ -98,6 +101,16 @@ if (analyticsAppKey) {
     analytics.addHandler(new SegmentHandler(deviceId, analyticsAppKey));
 }
 
+window.addEventListener(
+    'beforeunload',
+    event => {
+        store.dispatch(disconnectSpotTvRemoteControl(event)).catch(error => {
+            console.error('Failed to disconnect Spot TV on unload', error);
```

Comment:
Why is this catch for `disconnectSpotTvRemoteControl` in index.js but the catch for `disconnectFromSpotTV` is in actions.js?

Reply:
because one returns a promise and the other one does not, but maybe I can clean this up